src/components/augmentedReality/AugmentedReality.js

```diff
@@ -14,16 +15,15 @@ import { Wrapper, WrapperRow } from '../Wrapper';
 import { ARModal } from './ARModal';
 import { ARObjectList } from './ARObjectList';
 
-export const AugmentedReality = ({ navigation, onSettingsScreen }) => {
-  const [isModalVisible, setIsModalVisible] = useState(false);
-
-  const { data: staticData, loading, refetch } = useStaticContent({
-    name: 'arDownloadableDataList',
+export const AugmentedReality = ({ navigation, onSettingsScreen, tourID }) => {
+  const { data: staticData, error, loading, refetch } = useStaticContent({
+    name: `arDownloadableDataList-${tourID}`,
     type: 'json'
   });
 
   const [data, setData] = useState([]);
   const [isLoading, setIsLoading] = useState(loading);
+  const [isModalVisible, setIsModalVisible] = useState(false);
```

Comment:
how come the modal state here?

Reply:
It's just been moved. I thought it would be better to keep all the states together.

Reply:
i see